plugins/CoreHome/vue/src/ReportingMenu/ReportingMenu.vue

```diff
@@ -93,14 +100,21 @@
     </ul>
     <ul
       id="mobile-left-menu"
-      class="sidenav hide-on-large-only"
+      class="sidenav sidenav--reporting-menu-mobile hide-on-large-only"
```

Comment:
Any reason for adding a new class name here instead of using the existing ones or maybe the id? Just wondering why the already available attributes shouldn't be enough to select the element in CSS. If the available classes aren't unique enough you could simply use the class `reportingMenu` of the surrounding div.

Also the double-dash class name looks really ugly. I know we have a couple other places where that is used, but imho we should maybe try to get rid of them instead of adding new ones.

Reply:
Thanks for the comment @sgiehl!

Probably didn't need the new class, that's true. There's so many different ways of targeting elements in CSS. I don't think using the ID is the right way and I thought the sidenav wasn't unique to this element, and a combination of e.g. `.sidenav.hide-on-large-only` is also not the prettiest.
The `--` double dash is a modifier syntax from the BEM methodology. Although there might be a couple of similar uses in other places, it's true it wasn't necessary here either.

We should perhaps standardise how we do CSS and front-end naming in general and then we can have a common and shared approach to new additions, otherwise it'll be a mix of things.